src/AdoNetCore.AseClient/Internal/TypeMap.cs

```diff
@@ -136,7 +136,8 @@ private static int GetStringFormatLength(AseParameter parameter, Encoding enc, o
             switch (value)
             {
                 case string s:
-                    return Encoding.Unicode.GetByteCount(s);
+                    var count = Encoding.Unicode.GetByteCount(s);
```

Comment:
Pedant mode: why?

Reply:
was there for debugging, I have reverted it back now 👍